.github/workflows/ci.yml

```diff
@@ -299,3 +299,42 @@ jobs:
         run: pip install . -c ci/constraints-test/py310-min-install.txt -r reqs/requirements-dev-test.txt
       - name: Run the tests
         run: invoke tests --unit
+
+  import_gx:
+    needs: [unit-tests, static-analysis]
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.8"
```

Comment:
We should do this over all the python versions. This is what happens in the release pipeline. We can restrict this to 3.8 for the PR pipeline but it might be worth doing all versions here.